README.md

```diff
@@ -86,6 +86,8 @@ Lists.methods.makePrivate = new Method({
 });
 ```
 
+The `schema` option can be set to any object that has a `validator` function which, when called with no arguments, returns a validation function to use as the `validate` option (see below). SimpleSchema version 1.4+ has such a function.
```

Comment:
Do we need a `schema` option in this world at all? I think it's simpler to just have one API rather than two, one of which relies on an implicit contract between this object and another.